spec/spec_helper.rb

```diff
@@ -188,12 +188,6 @@ def setup_runtime_log
 
   config.raise_errors_for_deprecations!
 
-  # sometimes stuff hangs -> do not hang everything
-  config.include(Module.new {def test_timeout;30;end })
-  config.around do |example|
-    Timeout.timeout(test_timeout, &example)
-  end
-
```

Comment:
I def want that ... just set the timeout to something high for the slow tests

Reply:
Travis already has built-in timeout detection, not sure about Appveyor. I'll add it back anyways.